From bb406dd159479f4ef5fb7b4f11aecda093ae14e4 Mon Sep 17 00:00:00 2001
From: Heinrich Schuchardt <heinrich.schuchardt@canonical.com>
Date: Fri, 4 Jul 2025 09:05:35 +0200
Subject: [PATCH 411/534] eswin/ai_driver: type mismatch printing
 npu_info->peer_type

npu_info->peer_type is of type u64 (long long unsigned int). For printing
we must use %lld.

Signed-off-by: Heinrich Schuchardt <heinrich.schuchardt@canonical.com>
---
 drivers/soc/eswin/ai_driver/npu/parse_dep.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/soc/eswin/ai_driver/npu/parse_dep.c b/drivers/soc/eswin/ai_driver/npu/parse_dep.c
index 5a09746e08dc..c097ef2af355 100644
--- a/drivers/soc/eswin/ai_driver/npu/parse_dep.c
+++ b/drivers/soc/eswin/ai_driver/npu/parse_dep.c
@@ -230,7 +230,7 @@ static void dependency_consumer2producer(struct win_executor *executor,
 		    task->op_desc[i].event_op.submodel_type == E31)
 		{
 			npu_info->peer_type = E31;
-			dla_debug("e31 event op:%d,depcnt:%d,peer_type:%d\n", i,
+			dla_debug("e31 event op:%d,depcnt:%d,peer_type:%lld\n", i,
 			           dependency_count[i], npu_info->peer_type);
 		}
 
-- 
2.53.0

